### ISSUE TITLE
Archive project

### ISSUE THREAD

**Issue by steakunderscore** [rakyll/drive#68]
I think it would be useful to archive this project. The intent is to clearly signal to internet wanders that this project now lives at [odeke-em/drive](https://github.com/odeke-em/drive).

Docs: https://help.github.com/en/articles/archiving-repositories

**Comment by Berkmann18** [comment]
Yeah, this way most issues and stuff are directed at the right place (especially given that @odeke-em's fork is well... Still a fork).

**Comment by rakyll** [comment]
Doing it now, thanks for suggesting this.
